onnxruntime/core/platform/windows/env.cc

```diff
@@ -866,11 +866,19 @@ WindowsEnv::WindowsEnv() {
   InitializeCpuInfo();
 }
 
+
 /*
 Discover all cores in a windows system.
 Note - every "id" here, given it be group id, core id, or logical processor id, starts from 0.
 */
 void WindowsEnv::InitializeCpuInfo() {
+  // Initialize cpuinfo once on Windows similar to PosixEnv constructor.
```

Comment:
Do we nee a macro here if cpuinfo supported?

Reply:
I added code to force the library be available on Windows.